pytorch_lightning/loggers/comet.py

```diff
@@ -98,6 +106,7 @@ def __init__(self,
                  rest_api_key: Optional[str] = None,
                  experiment_name: Optional[str] = None,
                  experiment_key: Optional[str] = None,
+                 online: bool = True,
```

Comment:
You set `online` here instead of `offline`. It's not relevant but just to make consistency with other loggers since other loggers(Wandb, Neptune) have `offline: bool = False`.

Reply:
Ah, I haven't looked at the other loggers so I wasn't aware of that. Thanks for point it out; I'll change it now.